platform/javascript/os_javascript.cpp

```diff
@@ -248,11 +248,17 @@ static EM_BOOL _wheel_callback(int event_type, const EmscriptenWheelEvent *wheel
 
 	// Different browsers give wildly different delta values, and we can't
 	// interpret deltaMode, so use default value for wheel events' factor
+	int mask = _input->get_mouse_button_mask();
+	int button_flag = 1 << (ev->get_button_index() - 1);
 
 	ev->set_pressed(true);
+	mask |= button_flag;
+	ev->set_button_mask(mask);
```

Comment:
`set_button_mask` is already called once in line 229, remove that original invocation since we're calling it here.